readme.md

```diff
@@ -40,16 +40,29 @@ ln -s dev-lib/.editorconfig . && git add .editorconfig
 cp dev-lib/.jshintignore . && git add .jshintignore # don't use symlink for this
 ```
 
-For ESLint, you'll also likely want to make `eslint` as a dev dependency for your NPM package:
+It is a best practice to install the various tools as dependencies in the project itself, pegging them at specific versions as required. This will ensure that the the tools will be repeatably installed across environments. When a tool is installed locally, it will be used instead of any globally-installed version. To install packages locally, for example:
 
 ```bash
 npm init # if you don't have a package.json already
-npm install --save-dev eslint
+npm install --save-dev eslint jshint jscs grunt-cli
 git add package.json
 echo 'node_modules' >> .gitignore
+
+composer init # if you don't have a composer.json already
+composer require --dev "wp-coding-standards/wpcs=*"
+echo 'vendor' >> .gitignore
+
 git add .gitignore
 ```
 
+Also make sure you configure your `composer.json` to automatically add WPCS to the `installed_paths` for PHPCS:
+
+```json
+  "scripts": {
+    "post-install-cmd": [ "phpcs --config-set installed_paths ../../wp-coding-standards/wpcs/" ]
```

Comment:
It could be done inside of `install_tools` Bash function perhaps, but still it seems like the ideal would be for it to be done explicitly in the `composer.json`.